src/common/hwio.h

```diff
@@ -16,15 +16,30 @@
 #define HWIO_CLS_FAN    0x10 //class fan controller
 #define HWIO_CLS_HEATER 0x11 //class heater controller
 
+//pwm outputs
+#define HWIO_PWM_HEATER_BED 0 //BED PWM
+#define HWIO_PWM_HEATER_0   1 //NOZZLE PWM
+#define HWIO_PWM_FAN1       2 //PRINT FAN?
+#define HWIO_PWM_FAN        3 //NOZZLE FAN?
+
 #ifdef __cplusplus
 extern "C" {
 #endif //__cplusplus
 
+//a3ides analog inputs
+enum {
+    ADC_HW_IDENTIFY,    //!< PA3 - chan 3
+    ADC_TEMP_BED,       //!< PA4 - chan 4
+    ADC_TEMP_2,         //!< PA5 - chan 5
+    ADC_TEMP_HEATBREAK, //!< PA6 - chan 6
+    ADC_TEMP_0,         //!< PC0 - chan 10
+    ONE_BEHIND_LAST_ADC,
+} typedef Adc;
```

Comment:
/very subjective/
For some reason, I hate the look of `Adc`. Can it be `ADC`? :D

Reply:
It can't be ADC, it would look like macro. But ADC_t would be possible.

Reply:
Yes, good point!